port/common/omrsysinfo.c

```diff
@@ -163,6 +163,40 @@ omrsysinfo_get_processor_feature_name(struct OMRPortLibrary *portLibrary, uint32
 	return rc;
 }
 
+/**
+ * Generate the corresponding string literals for the provided OMRProcessorDesc.
+ *
+ * @param[in] portLibrary The port library.
+ * @param[out] buff The cpu features output string.
+ * @param[in] BUFF_SIZE The size of the buff.
+ */
+void
+omrsysinfo_print_processor_features(struct OMRPortLibrary *portLibrary, OMRProcessorDesc *desc, char * buff, const size_t BUFF_SIZE)
```

Comment:
Looking at omrport.h many of the similar type APIs name the two arguments `buffer` and `length`. I would say we follow suit and have some consistency here.